cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/common/PackageProcessor.java

```diff
@@ -20,7 +20,7 @@ public class PackageProcessor implements IPackageProcessor {
     public PackageProcessor(Repository repository) {
         this.repository = repository;
         this.fhirVersion = this.repository.fhirContext().getVersion().getVersion();
-        packageVisitor = new PackageVisitor(this.repository);
+        packageVisitor = new PackageVisitor(this.repository, null);
```

Comment:
The visitor has a constructor with just the repository.  No need to add all these null parameters in.

Reply:
fixed